### PR TITLE
Fixed not showing shipping estimate if already has postalCode

### PR DESCRIPTION
**Pull request by omninando**
#### How should this be manually tested?

1. Add [items to cart](https://fernando--checkoutio.myvtex.com/cart/add?&workspace=fernando&sku=298&qty=1&seller=1&sc=1);
2. Estimate shipping and refresh;
3. It should show calculated estimate.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by jeffersontuc — DISMISSED** [review]

File: react/components/EstimateShipping.tsx [hunk 1]
```diff
@@ -7,9 +7,13 @@ import { newAddress } from '../utils/address'
...
 const { AddressContainer, AddressRules, StyleguideInput } = components
 const { addValidation, removeValidation } = helpers
+interface InsertAddressResult {
+  success: boolean
+  orderForm: any
+}
 
 interface CustomProps {
-  insertAddress: (address: CheckoutAddress) => void
+  insertAddress: (coupon: CheckoutAddress) => Promise<InsertAddressResult>
```
Comment (jeffersontuc) [inline react/components/EstimateShipping.tsx]: Shouldn't it be `address` instead of `coupon`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Showing shipping estimate if has address postalCode;
```
Comment (jeffersontuc) [inline CHANGELOG.md:12]: Nit: `.`

File: react/components/EstimateShipping.tsx (hunk 1 above)
Comment (omninando) [reply react/components/EstimateShipping.tsx]: Done!

File: CHANGELOG.md (hunk 2 above)
Comment (omninando) [reply CHANGELOG.md:12]: If there is more than one topic I prefer to use the semicolon.

**Review by jeffersontuc — APPROVED** [review]

**Pull request merged**